frontend/src/components/verticalPodAutoscaler/VPAList.stories.tsx

```diff
@@ -1,4 +1,4 @@
-import { Meta } from '@storybook/react';
+import { Meta } from '@storybook/react/types-6-0';
```

Comment:
From the logs of the failing tests looks like this change is the issue. Either use the old package itself or include the changes to package.json also in the commit.